### DISCUSSION TITLE
Turn off Pull Request discovery

### DISCUSSION THREAD

**Discussion by jamers99** [rfennell/AzurePipelines#1174] [Q&A] [answered]
Is there a way to to turn off the PR search, I don't use that functionality in my template.

**Comment by rfennell** [comment, marked as answer]
No at this time there is not. If it is of use please raise [a new feature request](https://github.com/rfennell/AzurePipelines/issues/new?assignees=&labels=enhancement&template=feature_request.yaml) or consider submitting a PR with the code change

**Reply by rfennell** [reply]
This has been added on #1230
